api/discovery/discovery.proto

```diff
@@ -62,13 +62,16 @@ message StartDiscoveryRequest { optional string resource_group = 1; }
 message StartDiscoveryResponse { bool successful = 1; }
 
 message QueryRequest {
```

Comment:
While we are at it, we could probably rename this to something proper like `ListResourcesRequest`